cartography/cli.py

```diff
@@ -685,6 +693,13 @@ def main(self, argv: str) -> int:
         else:
             config.semgrep_app_token = None
 
+        # CVE feed config
+        if config.cve_api_key_env_var:
+            logger.debug(f"Reading NVD CVE API key environment variable {config.cve_api_key_env_var}")
```

Comment:
Leaving API Keys in logs is not a good security practice. I suggest either to remove the key completely or at least remove / hide most of the characters.

Reply:
Agree. On this case it's not an API key that is being logged, but the ENV_VAR holding it to follow the same pattern from previous API key configurations.